appium/tests/specs/live/inbox/CheckMessageProcessingErrors.spec.ts

```diff
@@ -29,12 +29,16 @@ describe('INBOX: ', () => {
     const wrongChecksumEmail = CommonData.wrongChecksumEmail.senderEmail;
     const wrongChecksumText = CommonData.wrongChecksumEmail.message;
 
-    //  Const for not integrity protected message BUG:https://github.com/FlowCrypt/flowcrypt-ios/issues/1144
-    // const notIntegrityProtectedSubject = CommonData.notIntegrityProtected.subject;
-    // // const notIntegrityProtectedEmail = CommonData.notIntegrityProtected.senderEmail;
-    // // const notIntegrityProtectedText = CommonData.notIntegrityProtected.message;
-    // const notIntegrityProtectedEncryptionBadge = CommonData.notIntegrityProtected.encryptionBadgeText;
-    // const notIntegrityProtectedSignatureBadge = CommonData.notIntegrityProtected.signatureBadgeText;
+    const notIntegrityProtectedSubject = CommonData.notIntegrityProtected.subject;
+    const notIntegrityProtectedEmail = CommonData.notIntegrityProtected.senderEmail;
+    const notIntegrityProtectedText = CommonData.notIntegrityProtected.message;
+
+    const keyMismatchUnexpectedlySugject = CommonData.keyMismatchUnexpectedly.subject;
+    const keyMismatchUnexpectedlyEmail = CommonData.keyMismatchUnexpectedly.senderEmail;
+    const keyMismatchUnexpectedlyText = CommonData.keyMismatchUnexpectedly.message;
+    const keyMismatchUnexpectedlyEncryptedBadge = CommonData.keyMismatchUnexpectedly.encryptedBadgeText;
+    const keyMismatchUnexpectedlySignatureBadge= CommonData.keyMismatchUnexpectedly.signatureBadgeText;
+    const firstAttachmentName = CommonData.keyMismatchUnexpectedly.firstAttachmentName;
```

Comment:
Here property names should be updated too - `keyMismatchSubject`, `keyMismatchEmail`...